src/controllers/user.controller.js

```diff
@@ -162,16 +159,93 @@ class UserController {
             res,
             200,
             null,
-            'Your account has been activated.'
+            "Your account has been activated."
           );
         }
       }
 
-      return responseGenerator.sendError(res, 400, 'Invalid validation token.');
+      return responseGenerator.sendError(res, 400, "Invalid validation token.");
     } catch (error) {
       return responseGenerator.sendError(res, 500, error.message);
     }
   }
+
+  /**
+   * @static
+   * @param {object} req - express request object
+   * @param {object} res - express response object
+   * @returns {object}- returns information about the follow status of the requested user
+   * @memberof UserController
+   */
+  static async followUser(req, res) {
+    try {
+      //  Please inform team to change the id of the decrypted user to id when been parsed with req object
+      const { followeeId } = req.body;
```

Comment:
I think the followee id should come from req.params and not from the request body; you would not want the user to typed in their userId

Reply:
Thank you for your insight, I will implement the changes.